### PR TITLE
New serverless pattern - appsync-eventbridge

### PR DESCRIPTION
**Pull request by shansv**
Description of changes:

This sample project demonstrates how to integrate Amazon EventBridge directly with AWS Appsync. The AWS AppSync resolver function request and response used with the EventBridge data source allows you to send custom events to the Amazon EventBridge bus.

https://docs.aws.amazon.com/appsync/latest/devguide/resolver-reference-eventbridge-js.html

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: appsync-eventbridge-datasource/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/appsync-eventbridge-datasource
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+- [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+- [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
```
Comment (mavi888) [inline appsync-eventbridge-datasource/README.md:14]: this is using CDK

File: appsync-eventbridge-datasource/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   cdk bootstrap --profile test 1111111111/us-east-1
+   ```
+5. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the `lib/cdk-stack.ts` file:
+   ```bash
+   npm run build && cdk deploy
+   ```
+
+## How it works
+
+Let's submit mutation request through AWS Appsync Console. Appsync valiate the schema and publish the message directly to Event Bus using the Appsync JS Resolver.
```
Comment (mavi888) [inline appsync-eventbridge-datasource/README.md:43]: be more specific with the tests
all tests need to be done as CLI if possible, so please add command line examples or screenshots and step by step on how to test and how to see the results.

File: appsync-eventbridge-datasource/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Delete stack
+
+```bash
+cdk destroy
+```
+
+---
+
+Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (mavi888) [inline appsync-eventbridge-datasource/README.md:53]: 23 :)

File: appsync-eventbridge-datasource/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Appsync to EventBridge Datasource 
+
+This pattern demonstrates sending events directly to Event Bus by integrating Appsync with Eventbridge Datasource.
```
Comment (mavi888) [inline appsync-eventbridge-datasource/README.md:3]: there is an image in the project that is not used in the readme. Maybe you can add it

File: appsync-eventbridge-datasource/src/app.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/*! Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (mavi888) [inline appsync-eventbridge-datasource/src/app.js:1]: where this is used in the code?

File: appsync-eventbridge-datasource/README.md (hunk 1 above)
Comment (shansv) [reply appsync-eventbridge-datasource/README.md]: Updated the Readme file

File: appsync-eventbridge-datasource/README.md (hunk 2 above)
Comment (shansv) [reply appsync-eventbridge-datasource/README.md]: Updated the Readme file

File: appsync-eventbridge-datasource/README.md (hunk 4 above)
Comment (shansv) [reply appsync-eventbridge-datasource/README.md]: Added architecture diagram to readme

File: appsync-eventbridge-datasource/src/app.js (hunk 5 above)
Comment (shansv) [reply appsync-eventbridge-datasource/src/app.js]: removed this folder

**Comment by mavi888** [comment]
Thanks for submitting this PR.
You can find the live pattern here:  https://serverlessland.com/patterns/appsync-eventbridge-datasource

**Pull request merged**